packages/plugins/rubygem-foreman_bootdisk/rubygem-foreman_bootdisk.spec

```diff
@@ -74,6 +74,8 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
+%exclude %{gem_instdir}/.tx
+%exclude %{gem_instdir}/gemfile.d
```

Comment:
This shouldn't be a thing because it explicitly lists the files that are packaged and these aren't included: https://github.com/theforeman/foreman_bootdisk/blob/9792de1e35c3682af97b5eaabf333935ca508a91/foreman_bootdisk.gemspec#L25-L27